tests/unit_tests/layout/test_layout_array.py

```diff
@@ -205,7 +205,7 @@ def test_build_layout(layout_array_north_four_LST_instance, tmp_test_directory,
     layout.export_telescope_list(crs_name="corsika")
 
     # Building a second layout from the file exported by the first one
-    layout_2 = LayoutArray(layout.telescope_list_file, name="test_layout_2")
+    layout_2 = LayoutArray(site=layout.site, mongo_db_config=db_config, name="test_layout_2")
```

Comment:
I dont think this is the correct way. Here we are generating the second layout instance based on an attribute from the first layout instance, which is not the idea of the test. The idea is generating a second class instance based on a telescope file. But I agree that in order to do that we have to be able to get the site from the telescope file (as discussed in #436). I propose we return this to how it was and comment it out until we have this new feature ready (and make sure we dont forget to uncomment later).

Reply:
This was a mistake. I changed it for a second to test something and forgot to change it back (my docker was broken so I had to run the tests using GitHub). I changed it back now, specifying the attribute name. 

To clarify, at the moment we don't need to have the site in the init if we provide a file in the init. We currently assume that the file contains the CORSIKA location and altitude. As part of #436 we should add a check that this is really true and if not read it from the DB.

Reply:
Ok, that sounds good. I think I understand it now. we only demand a site parameter in `_from_corsika_file_to_dict`, which is specifically used in case the telescope file is not passed. that is fine by now. we will have to make sure to set the self.site parameter from the telescope file in #436, but I agree we can chat a bit before implementing anything there.